Gopkg.toml

```diff
@@ -80,9 +80,13 @@ required = [
   name = "github.com/giantswarm/errors"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
   name = "github.com/giantswarm/helmclient"
 
+[[constraint]]
+  branch = "kubernetes-1.15.4"
```

Comment:
To be set to master once https://github.com/giantswarm/kubeconfig/pull/12 gets merged.